### PR TITLE
feat: add arrow upward icon

### PR DESCRIPTION
**Pull request by Spring3**
# Purpose of PR

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/components/icons/src/ArrowUpward.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { Fragment } from 'react';
+import { generateIcon } from '@contentful/f36-icon';
+
+export const ArrowUp = /*#__PURE__*/ generateIcon({
```
Comment (denkristoffer) [inline packages/components/icons/src/ArrowUpward.tsx:4]:
```suggestion
export const ArrowUpward = /*#__PURE__*/ generateIcon({
```

File: packages/components/icons/src/ArrowUpwardTrimmed.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Fragment } from 'react';
+import { generateIcon } from '@contentful/f36-icon';
+
+export const ArrowUpTrimmed = /*#__PURE__*/ generateIcon({
```
Comment (denkristoffer) [inline packages/components/icons/src/ArrowUpwardTrimmed.tsx:4]:
```suggestion
export const ArrowUpwardTrimmed = /*#__PURE__*/ generateIcon({
```

**Review by denkristoffer — COMMENTED** [review]
Looks like the build is failing because of naming issues. We also have to add the new exports to `packages/components/icons/src/index.ts`

File: packages/components/icons/src/ArrowUpwardTrimmed.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export const ArrowUpwardTrimmed = /*#__PURE__*/ generateIcon({
+  name: 'ArrowUpwardTrimmed',
+  path: (
+    <Fragment>
+      <path d="M21.65 40.55V16.45L10.8 27.35L7.45 24L24 7.45L40.55 24L37.25 27.35L26.35 16.45V40.55Z"></path>
+    </Fragment>
+  ),
+  trimmed: true,
+  viewBox: '0 0 48 48',
```
[the remark this answers is not in the document]
Comment (Spring3) [reply packages/components/icons/src/ArrowUpwardTrimmed.tsx:12]: What is the "regular" icon size?

[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/components/icons/src/ArrowUpwardTrimmed.tsx]: I mean the `ArrowUpward.tsx` you're adding. Trimmed is for cases where you have an icon with a view box smaller than 24x24 and you want a variant without the extra padding. I know this is not clear from the docs 🙃

**Review by denkristoffer — APPROVED** [review]
Great, thanks!

**Pull request merged**